src/librustc_typeck/check/regionck.rs

```diff
@@ -301,15 +302,15 @@ impl<'a, 'gcx, 'tcx> RegionCtxt<'a, 'gcx, 'tcx> {
     /// `intravisit::Visitor` impl below.)
     fn visit_fn_body(
         &mut self,
-        id: ast::NodeId, // the id of the fn itself
+        id: hir::HirId, // the id of the fn itself
         body: &'gcx hir::Body,
         span: Span,
     ) {
         // When we enter a function, we can derive
-        debug!("visit_fn_body(id={})", id);
+        debug!("visit_fn_body(id={:?})", id);
```

Comment:
Maybe you should add a Display impl for HirId?

Reply:
I'd probably prefer to do it in a separate PR, but sure, why not; how would you suggest it should look like?

Reply:
You can probably just call the `Debug` impl for now, and then just revert the changes to use `{:?}`